### PR TITLE
Adding AddressResolver

### PR DESCRIPTION
**Pull request by justinjmoses**

### CODE REVIEW

File: lib/abis/ropsten/AddressResolver.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import AddressResolver from '../mainnet/AddressResolver';
+export default AddressResolver;
```
Comment (evgenyboxer) [inline lib/abis/ropsten/AddressResolver.js]: In the exchange we usually do in one line - 
`export { default } from '../mainnet/AddressResolver';`

**Comment by evgenyboxer** [comment]
@justinjmoses lgtm (had one minor comment)

However, since its my first review for this library, I'll let @clementbalestrat have another look to approve it.

File: lib/abis/ropsten/AddressResolver.js (hunk 1 above)
Comment (justinjmoses) [reply lib/abis/ropsten/AddressResolver.js]: this is autogenerated in `tools/abitojs.js`, but we could probably fix that with not too much trouble

**Comment by justinjmoses** [comment]
This is out of date and superseded. Closing.

**Pull request closed without merging**